craft.lic

```diff
@@ -144,6 +144,7 @@ class Craft
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 6)
       stow_hands
+      walk_to(@training_room)
```

Comment:
💡 It really feels like we ought to have a helper method for this

Reply:
Any suggestions?  walk_to in itself is a helper method